frontend/src/components/projectDetail/styles.scss

```diff
@@ -21,3 +21,7 @@
 .menu-items-container::-webkit-scrollbar {
   display: none;
 }
+
+.react-tooltip#dueDateBoxTip {
```

Comment:
You probably intended to refer to `#dueDateBoxTooltip` instead of `#dueDateBoxTip`. This mismatch should have caused the tooltip to be below the container. 
![Screenshot from 2023-05-15 16-47-35](https://github.com/hotosm/tasking-manager/assets/51614993/1f989f93-4c01-430b-8ea3-e1f9328a4204)